CrowsZA/functions/fn_fireSupport.sqf

```diff
@@ -0,0 +1,205 @@
+/*/////////////////////////////////////////////////
+Author: Radkewolf
+			   
+File: fn_fireSupport.sqf
+Parameters: pos
+Return: none
+
+Sets off firesupport in the targeted area
+
+*///////////////////////////////////////////////
+params [["_pos",[0,0,0],[[]],3], ["_logic",objNull,[objNull]]];
+
+// Validate module target
+scopeName "Main";
+private _fnc_errorAndClose = {
+    params ["_msg"];
+    diag_log _msg;
+    breakOut "Main";
+};
+
+// open dialog
+//ZEN
+private _onConfirm =
+{
+	params ["_dialogResult","_in"];
+	_dialogResult params
+	[
+		"_type",
+		"_customType",
+		"_radius",
+		"_seconds",
+		"_delay",
+		"_salvos",
+		"_guns"
+	];
+
+	//Get in params again
+	_in params [["_pos",[0,0,0],[[]],3], ["_logic",objNull,[objNull]]];
+
+	//Check if placed on an object
+	_create = isNull _logic;
```

Comment:
any reason this is a global variable? Ideally we want to use ``private _create`` where we can.

Reply:
No reason apart from simply not knowing :D

Reply:
haha fair enough :D